prompts.js

```diff
@@ -62,8 +62,20 @@ Example: com.company.app`;
       {
         name: 'Advanced',
         value: 'advanced'
+      },
+      {
+        name: 'Custom',
+        value: 'custom'
       }
     ],
-    default: 'simple'
+    default: 'custom'
```

Comment:
I think the default value must be simple.

Reply:
done